lib/p5.play.js

```diff
@@ -1396,7 +1396,6 @@ function Sprite(pInst, _x, _y, _w, _h) {
       return;
     }
     this.setCollider('rectangle');
-    pInst.quadTree.insert(this);
```

Comment:
`setDefaultCollider` was calling `quadTree.insert`, which is 100% redundant because `setCollider` will _always_ call `quadTree.insert`.  The quadTree checks early to avoid letting any sprite be added twice so there wasn't much optimization here, so this is just cleanup.